src/commands/createNewProject/IProjectWizardContext.ts

```diff
@@ -29,6 +29,8 @@ export interface IProjectWizardContext extends IActionContext {
     openApiSpecificationFile?: Uri[];
 
     targetFramework?: string | string[];
+
+    dockerfile?: boolean;
```

Comment:
Property name is a bit odd for a boolean. If I see `dockerfile`, I think it's the dockerfile path or Uri. I would have called it `addDockerfile` or something like that.

But that being said, I don't think you need this property anymore. I'm assuming it used to be used to determine whether or not to add the `CreateDockerfileProjectStep` but we've covered that in another way.